x-pack/filebeat/module/infoblox/nios/test/generated.log-expected.json

```diff
@@ -0,0 +1,2476 @@
+[
+    {
+        "event.code": "openvpn-master",
+        "event.dataset": "infoblox.nios",
+        "event.module": "infoblox",
+        "event.original": "January 29 06:09:59 volup208.invalid eosquir: openvpn-master OpenVPN 1.5191 [igmp] [nulapari] mwritten",
+        "fileset.name": "nios",
+        "input.type": "log",
+        "log.offset": 0,
+        "network.protocol": "igmp",
+        "observer.product": "Network",
+        "observer.type": "IPAM",
+        "observer.vendor": "Infoblox",
+        "observer.version": "1.5191",
+        "rsa.db.index": "mwritten",
+        "rsa.internal.messageid": "openvpn-master",
+        "rsa.misc.event_source": "volup208.invalid",
+        "rsa.misc.version": "1.5191",
+        "rsa.time.day": "29",
+        "rsa.time.event_time": "2020-01-29T08:09:59.000Z",
+        "rsa.time.month": "January",
+        "service.type": "infoblox",
+        "tags": [
+            "infoblox.nios",
+            "forwarded"
+        ]
+    },
+    {
+        "event.code": "cloud_api",
+        "event.dataset": "infoblox.nios",
+        "event.module": "infoblox",
+        "event.original": "Feb 12 13:12:33 com1060.api.example 10.14.94.160 cloud_api[tur]: proxying request to atio5608.www5.localhost(10.202.204.154) eFini https://www.example.org/exe/iatu.jpg?orsitame=reprehe#rsitam ggp issusci",
```

Comment:
Doesn't look like we're capturing the first IP yet? Also, I'm wondering if it makes more sense to use the `source`/`destination` fields here rather than `host`?

Reply:
The issue with this logs is that it captures the first IP in a field called hhostip, which we're not mapping because it's not documented, maybe I could append that to `host.ip` same as the `hostip` field.

Reply:
So based off of the message and reading a bit on the cloud api stuff, it looks like the `hostip` field here (`10.202.204.154`) appears to be the IP that is receiving a request rather than the host that an event is occurring on or about some internals to the host (which is generally how I think about the `host` fields). I'm thinking that it probably makes sense more as either `destination` or `server` and maybe the `hhostip` is the originating server so `client` or `source`?

Either way, in the case of something like a proxy or load balancer i'd probably leave the `host.*` fields to represent the actual ip/mac/whatever other info of the proxy itself and fill in `source`/`destination` or `client`/`server` to describe the connections of the request. That way if someone wanted to see something like all the requests to a certain server from a given proxy they could do something like `host.name:myproxy.domain AND destination.ip:www.example.com`.

Is there a mechanism for generating some of these logs where we can figure out what the heck `hhostip` actually is?